docs/concepts/guide-data-apis.md

```diff
@@ -53,12 +53,27 @@ Here's the full list of operators supported:
 
 | op  |  Meaning  | Supported types | Examples  |
 |-----|-----------|----------|----------------|
-| eq  | Equals     | integer, string | project_id=eq.pr123 |
+| eq  | Equals     | integer, string, timestamp | project_id=eq.pr123 |
+| neq  | Not Equals     | integer, string, timestamp | amount=neq.531 |
 | lt  | Less than  | integer, string, timestamp | amount=lt.100 |
 | lte | Less than or equal to  | integer, string, timestamp | updated_at=lte.2020-06-01T00:00:00.000-08:00 |
 | gt | Greater than  | integer, string, timestamp | amount=gt.100 |
 | gte | Greater than or equal to  | integer, string, timestamp| updated_at=gte.2020-06-01T00:00:00.000-08:00 |
 | in | Is one of  | integer, string | id=in.(id1,id2,id3) |
+| not_in | Is not one of  | integer, string | creator_user_id=not_in.("usyKlnkmSVw9") |
+| is | Is equals  | null, boolean | is_reimbursable=is.true |
+| is_not | Is not equals  | null, boolean | is_reimbursable=is_not.true |
+| like | Pattern matches with case sensitive  | integer, string | cost_center->name=like.%porate% |
+| not_like | Pattern not matches with case sensitive  | integer, string | cost_center->name=not_like.%porate% |
+| ilike | Pattern matches by ignoring case sensitive  | integer, string | creator_user_id=ilike.%usyKlnkmsvw9% |
+| not_ilike | Pattern not matches by ignoring case sensitive  | integer, string | creator_user_id=not_ilike.%usyKlnkmsvw9% |
+| cs | Does it contains  | array, jsonb | category_ids=cs.[115260, 115257], cost_center=cs.{"code":"13597", "name": "Corporate", "id": 6632} |
+| csn | Does it contains or None  | restricted array, restricted jsonb | restricted_category_ids=csn.[115260, 115257], restricted_cost_center=csn.{"code":"13597", "name": "Corporate", "id": 6632} |
```

Comment:
the client will be passing json input hence `null` instead of `None`
 `contains or is null` sounds more clean